lib/tasks/import.rake

```diff
@@ -0,0 +1,91 @@
+require 'csv'
+
+namespace :csv_load do
+  task customers: :environment do
+    CSV.foreach('./db/data/customers.csv', headers: true) do |row|
+      Customer.create!(row.to_hash)
+    end
+  end
+
+  task invoice_items: :environment do
+    CSV.foreach('./db/data/invoice_items.csv', headers: true) do |row|
+      if row[5] == 'pending'
+        status = 0
+      elsif row[5] == 'packaged'
+        status = 1
+      elsif row[5] == 'shipped'
+        status = 2
+      end
+      InvoiceItem.create!({
+        :id => row[0],
+        :item_id => row[1],
+        :invoice_id => row[2],
+        :quantity => row[3],
+        :unit_price => row[4],
+        :status => status,
+        :created_at => row[6],
+        :updated_at => row[7]
+      })
+    end
+  end
+
+  task invoices: :environment do
+    CSV.foreach('./db/data/invoices.csv', headers: true) do |row|
+      if row[2] == 'in progress'
+        status = 0
+      elsif row[2] == 'completed'
+        status = 1
+      elsif row[2] == 'cancelled'
+        status = 2
+      end
+      Invoice.create!({
```

Comment:
Just to make sure I'm understanding this correctly, the Invoice.create! is making 1 Invoice object with the entire csv data?

Reply:
The each at the top of this task means that it will execute the block on one row after another. The create on that line is making a single Invoice using a single row of the CSV file. The CSV files are about 1000 rows long.